packages/blocks/github-pr-overview/src/dev.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * This is the entry point for developing and debugging.
+ * This file is not bundled with the block during the build process.
+ */
+import React from "react";
+import ReactDOM from "react-dom";
+
+import { MockBlockDock } from "mock-block-dock";
+
+import Component from "./index";
+import exampleGraph from "../example-graph.json";
+
+const node = document.getElementById("app");
+
+const App = () => (
+  <MockBlockDock
+    initialEntities={exampleGraph.entities}
+    initialEntityTypes={exampleGraph.entityTypes}
+    initialLinks={exampleGraph.links}
```

Comment:
`yarn dev` now allows for testing this without loading HASH

Reply:
Just tested it, awesome!!